src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocLexer.g4

```diff
@@ -253,9 +251,7 @@ mode value;
 Leading_asterisk2: LEADING_ASTERISK -> type(LEADING_ASTERISK);
 Space6: WS -> type(WS);
 Newline4: NEWLINE -> type(NEWLINE);
-Package2: PACKAGE -> type(PACKAGE);
-Dot2: DOT -> type(DOT);
-Class2: CLASS -> type(CLASS);
```

Comment:
please explain Dot2 and Class2 removals.

Reply:
Same reason as for `@see` tag. [`@value`](http://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#value) reference seems to be same as `@see` tag reference. 
In a reference we can't separate a package from a class without extra context about that reference. 

`DOT` gets recognized in `PACKAGE_CLASS` itself, so no need to tokenize it separately right ?

Reply:
ok